src/Libraries/Microsoft.Extensions.Http.Diagnostics/README.md

```diff
@@ -18,6 +18,99 @@ Or directly in the C# project file:
 </ItemGroup>
 ```
 
+## Usage Example
+
+### HTTP Client Logs Enrichment and Redaction
+
+These components enable enriching and redacting HTTP Client request logs. They remove built-it HTTP Client logging.
```

Comment:
```suggestion
These components enable enriching and redacting `HttpClient` request logs. They remove built-it HTTP Client logging.
```

Reply:
Only HttpClient instances created with the IHttpClientFactory benefit from these features, correct?

Reply:
Unfortunately, yes.

Reply:
Applied the suggestion, thanks!